examples/apps/dashboard/dashboard.py

```diff
@@ -0,0 +1,227 @@
+        {images_html}
+    </div>
+    '''
+    return wrapper_html
+
+def to_styles_projects_table(data: pd.DataFrame):
+    data["Company"]=data.apply(generate_company_html, axis=1)
+    data["Completion"]=data.apply(generate_progress_bar, axis=1)
+    data["Members"]=data.apply(generate_member_images, axis=1)
+    data = data.drop(columns=["CompanyImage"])
+    styled_df = data.style
+    styled_df = (
+        styled_df.hide(axis="index")
+        .set_table_styles(
+            [
+                {
+                    "selector": "th",
+                    "props": [
+                        ("background-color", "white"),
+                        ("font-weight", "bold"),
+                        ("border-bottom", "1px solid black"),
+                        ("text-align", "left"),
+                    ],
+                },
+                {
+                    "selector": "td",
+                    "props": [
+                        ("padding", "10px"),
+                        ("background", "white"),
+                        ("border-top", "1px solid black"),
+                    ],
+                },
+            ]
+        )
+        .set_properties(
+            **{"text-align": "left"},
+        )
+    )
+
+    return styled_df
+
+
+with pn.config.set(sizing_mode="stretch_width"):
+    web_site_views = pn.Column(pn.Column(
+        "#### Website Views\n\nLast Campaign Performance",
+        pn.pane.ECharts(
+            get_website_views_config(),
+            height=250,
+            margin=(0, 5),
+        ),
+        pn.Spacer(sizing_mode="stretch_height"),
+        pmu.Divider(margin=(10,10,-5,10)),
+        last_update("campaign sent 2 days ago"),
+    ), styles=PAPER_STYLES, height=400, margin=10, )
+
+    daily_sales = pn.Column(
+        "#### Daily Sales\n\n**+15%** increase in todays sales",
+        pn.pane.ECharts(
+            get_daily_sales_config(), height=250, margin=(0, 5),
+        ),
+        pn.Spacer(sizing_mode="stretch_height"),
+        pmu.Divider(margin=(10,10,-5,10)),
+        last_update("updated 4 min ago"),
+        styles=PAPER_STYLES,
+        margin=10,
+        height=400,
+    )
+    completed_tasks = pn.Column(
+        "#### Completed Tasks\n\nLast Campaign Performance",
+        pn.pane.ECharts(
+            get_completed_tasks_config(),
+            height=250,
+            margin=(0, 5),
+        ),
+        pn.Spacer(sizing_mode="stretch_height"),
+        pmu.Divider(margin=(10,10,-5,10)),
+        last_update("just updated"),
+        styles=PAPER_STYLES,
+        margin=10,
+        height=400,
+    )
+
+    plots = pn.Row(web_site_views, daily_sales, completed_tasks, sizing_mode="stretch_both")
+    project_table = pn.Column(
+        "#### Projects\n\n**30 done** this month",
+        pn.panel(to_styles_projects_table(get_project_data()), sizing_mode="stretch_width"),
+        styles=PAPER_STYLES,
+        margin=10,
+        height=550,
+    )
+    timeline_config = [
```

Comment:
Actually thinking about it, I would prefer not to ship `Timeline` for the time being. It's still a `mui/lab` component which means it's not stable and it's also not a must have imo.

Reply:
Would it be OK to introduce a `.lab` module with the same purpose as the Material UI lab concept? And then put the `Timeline` there? 

Its a common thing. For example Streamlit has `streamlit.experimental` module.

Reply:
I've moved the `Timeline` out of `panel_material_ui` to seperate example `MaterialUIComponent`. But now I have #190.